src/pages/Categories/CategoriesBeta.vue

```diff
@@ -173,6 +185,16 @@ export default {
 			return category.retinaImage?.url ?? '';
 		}
 	},
+	computed: {
+		contentGroups() {
+			return this.pageData?.page?.pageLayout?.contentGroups ?? [];
+		},
+		faqContentGroup() {
+			return this.contentGroups?.find(({ type }) => {
+				return type ? type === 'frequentlyAskedQuestions' : false;
```

Comment:
Could be simplified to `return type === 'frequentlyAskedQuestions'`

Reply:
> Could be simplified to `return type === 'frequentlyAskedQuestions'`

I thought this too -- this exact code is actually on several pages, and I probably wrote it and it was copied and pasted :)

Reply:
Yes, I was trying to stay consistent with the other FAQ code on the MonthlyGoodLandingPage.vue. Should I leave it? @eddieferrer